src/components/tracked-controls.js

```diff
@@ -145,12 +145,12 @@ module.exports.Component = registerComponent('tracked-controls', {
     // Compose pose from Gamepad.
     pose = controller.pose;
     // If no orientation, use camera.
-    if (pose.orientation) {
+    if (pose && pose.orientation) {
```

Comment:
Is it possible / correct for a pose to not have orientation? We may want to let it break to identify underlying issues instead of graceful handle the bogus case.